actions/action-install-fluvio-cluster.sh

```diff
@@ -10,16 +10,22 @@ curl -sSf https://raw.githubusercontent.com/infinyon/fluvio/master/install.sh |
 echo 'export PATH="$HOME/.fluvio/bin:$PATH"' >> $HOME/.bash_profile
 . $HOME/.bash_profile
 
-# Install Fluvio System Charts
-fluvio cluster start --setup --local --sys
+REPO_VERSION="$(curl -sSf https://raw.githubusercontent.com/infinyon/fluvio/master/VERSION)"
+CHART_VERSION="${REPO_VERSION}-${GITHUB_SHA}"
 
-# Run Fluvio Cluster Pre-Install Check
-
-fluvio cluster check
+#
+# Install Fluvio Cluster
+#
 
+# Install Local Fluvio Cluster
 if [ "$CLUSTER_TYPE" = "local" ]; then
-        # Install Local Fluvio Cluster
+
+    # If VERSION is equal to exactly "latest", use LATEST channel
+    if [ "${VERSION}" == "latest" ]; then
+        fluvio cluster start --rust-log $RUST_LOG --develop --local --spu $SPU_NUMBER --chart-version="${CHART_VERSION}"
+    else
         fluvio cluster start --rust-log $RUST_LOG --develop --local --spu $SPU_NUMBER
```

Comment:
is this what is expected from user?

Reply:
So at the end of the `publish.yml` step, a new helm chart called `fluvio-app:VERSION-HASH` is pushed to the chart museum. This corresponds to the new `latest` Fluvio CLI. However, `fluvio cluster start --local` has the VERSION it was compiled with baked-in (in this case it'd be `0.8.5`), and it tries to find a helm chart with just that `fluvio-app:VERSION`, which does not exist (because we have only run `publish.yml`, not `release.yml`).

By the time that the `0.8.5` CLI reaches the user, we will also have an `0.8.5` helm chart, at which point `--chart-version` will not be necessary. However, for the purpose of testing, we need to manually supply the helm chart version because otherwise `fluvio-cluster` will not be able to find it.

Reply:
Oh also to note is that the `--chart-version` flag is only provided when `VERSION=latest`, so the instance of this test that runs on `stable` will run this second command, with no `--chart-version` (i.e. using stable).

Reply:
I would expect `curl -fsS https://packages.fluvio.io/v1/install.sh | VERSION=latest bash` to be sufficient.

Reply:
Oh so all you want is to check the installer? Don't want to spin up a cluster to check on after installing?

Reply:
Yes but I would expect to use installer.  Right?

Reply:
Right so if I'm a brand-new user the very first thing I do is to use the `install.sh` one-liner. Then, if I was using fluvio locally I'd need to use `fluvio cluster start`. So it depends on how far through this flow we want to test here.

Reply:
We should perform exactly same action as user would do.   The action is described here: https://github.com/infinyon/fluvio/blob/master/actions/action-install-fluvio-cluster.sh

Reply:
You just linked to this file?

But also the setup here in `action-install-fluvio-cluster.sh` currently runs extra commands that I think a user would not typically run, such as `fluvio cluster start --sys` and `fluvio cluster check`. One of the things I was doing here was eliminating those lines so the setup flow is exactly the same as the user.

Reply:
Yes, I agree, we should remove them